Game/main.gd

```diff
@@ -0,0 +1,11 @@
+extends Node2D
+
+
+# Called when the node enters the scene tree for the first time.
+func _ready() -> void:
+	get_tree().change_scene_to_file("res://level.tscn")
```

Comment:
Please verify that this does not skip the main menu when you press the play button (not the play current scene button)

Reply:
it does not, I showed you me pressing the big triangle and it going to the menu